app/controllers/application_controller.rb

```diff
@@ -52,6 +52,7 @@ class ApplicationController < ActionController::Base
   include_concern 'TreeSupport'
   include_concern 'SysprepAnswerFile'
   include_concern 'ReportDownloads'
+  include_concern 'Show'
```

Comment:
Lookin at  this PR #10255, I think it makes more sense to wait until that PR is merged and used those methods (`show_performance`,`show_timeline`, etc..), they look very similar to these ones 

WDYT @abonas  @himdel  @martinpovolny